docs/gadget-devel/gadget-ebpf-api.md

```diff
@@ -530,10 +538,12 @@ int ig_execve_x(struct syscall_trace_exit *ctx)
 }
 ```
 
-## Kernel stack maps
+## Stack maps
+
+### Kernel stack traces
 
-To make use of kernel stack maps, gadgets must include
-[gadget/kernel_stack_map.h](https://github.com/inspektor-gadget/inspektor-gadget/blob/43d7b29f43d6ced34004ed20a7508c25cf6a5fb9/include/gadget/kernel_stack_map.h).
+To make use of kernel stack traces, gadgets must include
+[gadget/kernel_stack_map.h](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/include/gadget/).
```

Comment:
nit: use %IG_BRANCH% instead of main so it points to the right version?

(Also, the file named was stripped from the link, is it intended?)